Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+
+WORKDIR /home/node/app
+COPY package*.json ./
+COPY . .
+# EVN NODE_ENV=development
+# ENV NODE_PATH=./build
+RUN yarn add prisma -g
+
+EXPOSE 3000
+CMD ["sh", "-c", "yarn vercel-build ; yarn prisma db push; yarn start"]
```

Comment:
`yarn prisma db push` if we run `vercel-build` - everything prisma related is already done for us luckily, I think this could be safely removed!

Reply:
not if you start from a blank db